packages/react/src/analytics/integrations/shared/GoogleConsentMode/index.ts

```diff
@@ -0,0 +1,5 @@
+export {
+  GoogleConsentMode,
+  default as googleConsentMode,
```

Comment:
I think we can improve these namings as they are too similar to understand what each one represents.

Reply:
I follow the same pattern with Analytics/AnalyticsWeb (instances) namings, but I understand that it might be difficult to distinguish. Do you have any suggestions?